fish-in-the-sea/huxley-comics-inspector
# Feature/prevent too much requests

## What (v. 1.1.2)
The data used for the popup is now static instead of calculating the size of each state at runtime.

## How to test
Open the extension popup and the dashboard should show the latest stats about each issue.
Also, the query is much faster as it load way less data.
